IPython/extensions/autoreload.py

```diff
@@ -233,7 +238,7 @@ def filename_and_mtime(self, module):
 
         return py_filename, pymtime
 
-    def check(self, check_all=False, do_reload=True):
+    def check(self, check_all=False, do_reload=True, use_deduper_reload=True):
```

Comment:
Ok.

I'd like to see if there is a way to turn off use_duper_reload at runtime; just in case it breaks something maybe have the default as in instance varaible that cn be change with a parameter to the magic.

I'm og with true being the default, but scare we'd break someone without options to turn it off without changing version